### PR TITLE
Porting parts of gui-lib to Typed Racket

### PR DESCRIPTION
**Pull request by samth**
This is an initial pull request -- not ready to merge yet.

Work done by @earldean.

### CODE REVIEW

File: gui-lib/framework/private/Untitled.rkt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#lang racket
+
+(define specs null)
+
+(set! specs (cons (list "hello" "world") specs))
+
+(cons (list "hello" "world") specs)
```
Comment (stamourv) [inline gui-lib/framework/private/Untitled.rkt]: I think this file was included by mistake.

File: 0001-gui-utils.rkt-ported-to-typed-racket.patch [hunk 2]
```diff
@@ -0,0 +1,405 @@
+From 30df05652395718c82c59df6a9d92702c70dcb7b Mon Sep 17 00:00:00 2001
+From: Earl Dean <earldean@indiana.edu>
+Date: Wed, 21 Oct 2015 18:22:21 -0400
+Subject: [PATCH] gui-utils.rkt ported to typed racket
```
Comment (stamourv) [inline 0001-gui-utils.rkt-ported-to-typed-racket.patch]: I think those got added by mistake.

File: gui-lib/framework/private/Untitled.rkt (hunk 1 above)
Comment (samth) [inline gui-lib/framework/private/Untitled.rkt]: Yeah, this is not worth reviewing yet.

**Comment by stamourv** [comment]
The main concern I have is: will this affect the usability of the GUI system from untyped Racket? I'm worried about a repeat of the math library situation.

**Comment by samth** [comment]
I don't think we plan to merge this or keep working on it.

**Pull request closed without merging**